pubpub/pubpub
# update: add affiliation and self-uri to JATS exports

Adds author affiliations (requested offline by AAS) and self-uri to JATS in PubPub (resolves #2131). Along the way, fixes an issue with the dev worker nodemon command (resolves #2291) and bumps pandoc from 2.18 to 2.19.2, adding some LaTeX compatibility features.

There is a corresponding PR in pubpub-pandoc to bump the pandoc-types version accordingly, which should not be a breaking change (https://github.com/pubpub/prosemirror-pandoc/pull/20).

_Test plan_
- Create Pub with multiple authors, some with the same affiliation, some with multiple affiliations separated by semicolons (e.g. http://localhost:9876/pub/5ybxj2ct/draft)
- Export as JATS
- In exported JATS, make sure that authors have an xref tag that points to the correct aff tag, e.g. `<xref ref-type="aff" rid="aff-0"/>`
- In exported JATS, make sure that all affiliations are listed one time at the bottom of the contrib-group, with IDs that match the xrefs above, e.g.
```
<aff id="aff-0">
<institution-wrap>
<institution>Maximegalon University</institution>
</institution-wrap>
</aff>
```
- Make sure the pub has a self-uri tag at the bottom of article-meta that links to the Pub URI, e.g. `<self-uri xlink:href="https://demo.pubpub.org/pub/5ybxj2ct">This pub is available from https://demo.pubpub.org/pub/5ybxj2ct</self-uri>`
- Export to Word and PDF, and make sure the exports work properly
- Remove all the authors with affiliations and make sure the Pub exports to JATS, Word, and PDF properly.
